frontend/src/metabase/visualizations/lib/save-chart-image.ts

```diff
@@ -24,14 +26,17 @@ export const saveChartImage = async (selector: string, fileName: string) => {
     return;
   }
 
-  node.classList.add(SAVING_DOM_IMAGE_CLASS);
-
   const { default: html2canvas } = await import("html2canvas-pro");
   const canvas = await html2canvas(node, {
     useCORS: true,
-  });
+    onclone: (doc: Document, node: HTMLElement) => {
```

Comment:
I moved the logic inside onclone too, instead of adding and removing the classes, so that the style changes don't appear for a brief moment when the export is slow